docker-storage-setup.spec

```diff
@@ -54,9 +38,12 @@ install -p -m 755 %{SOURCE5} %{buildroot}/%{dsslibdir}/dss-child-read-write
 %files
 %{_unitdir}/docker-storage-setup.service
 %{_bindir}/docker-storage-setup
-%{dsslibdir}/docker-storage-setup
+%config(noreplace) %{_sysconfdir}/sysconfig/docker-storage-setup-override
 %config(noreplace) %{_sysconfdir}/sysconfig/docker-storage-setup
+%dir %{dsslibdir}
```

Comment:
Why do we need `%dir`. Following webpage seems to say, 

`if a directory is specified in the %files list, the contents of that directory, and the contents of every directory under it, will automatically be included in the package`

http://www.rpm.org/max-rpm-snapshot/s1-rpm-inside-files-list-directives.html

And we don't want that. We are mentioning each file individually.